gui/src/components/Taipy/Image.tsx

```diff
@@ -129,25 +129,35 @@ const Image = (props: ImageProps) => {
 
     return (
         <Tooltip title={hover || ""}>
-            <ImageButton
-                focusRipple={!!onAction}
-                style={style}
-                onClick={handleClick}
-                disabled={!active || !onAction}
-                className={className}
-                id={id}
-            >
-                <ImageSrc style={imgStyle} />
-                {onAction ? <ImageBackdrop className="MuiImageBackdrop-root" /> : null}
-                {label === undefined ? null : (
-                    <ImageSpan>
-                        <Typography component="span" variant="subtitle1" color="inherit" sx={imgSx} className={getSuffixedClassNames(className, "-label")}>
-                            {label}
-                            <ImageMarked className="MuiImageMarked-root" />
-                        </Typography>
-                    </ImageSpan>
-                )}
-            </ImageButton>
+            {onAction || label ? (
+                <ImageButton
+                    focusRipple={!!onAction}
+                    style={style}
+                    onClick={handleClick}
+                    disabled={!active || !onAction}
+                    className={className}
+                    id={id}
+                >
+                    <ImageSrc style={imgStyle} />
+                    {onAction ? <ImageBackdrop className="MuiImageBackdrop-root" /> : null}
+                    {label === undefined ? null : (
+                        <ImageSpan>
+                            <Typography
+                                component="span"
+                                variant="subtitle1"
+                                color="inherit"
+                                sx={imgSx}
+                                className={getSuffixedClassNames(className, "-label")}
+                            >
+                                {label}
+                                <ImageMarked className="MuiImageMarked-root" />
+                            </Typography>
+                        </ImageSpan>
+                    )}
+                </ImageButton>
+            ) : (
+                <img id={id} src={content} style={style} className={className} />
```

Comment:
So the potential svg is embedded in an img... hmm ok...

Reply:
Didn't think of this one ...

Reply:
Yeah I was not sure I read it well, but it tickled me as well...

Reply:
We could also handle content as svg text ie 
```
content = "<SVG> ...</SVG>"
<|image|content={content}|>
```
What do you think ?

Reply:
You meam, right in a GUI app, on the user side ? It's a bit dirty, don't you think ? SVG code can be messy, and putting that in your app's code may make it look like crap.

Reply:
just another option ... it can be loaded from a file or returned by a function ...

Reply:
OK, why not, then.